### PR TITLE
Feature/update for build and release

### PR DESCRIPTION
**Pull request by jakeberesford-palmetto**
## Pull request checklist

Before submitting your PR, please review the following checklist:

- [ ] Consider adding a unit test if your PR resolves an issue.
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

### Breaking changes

- [ ] Check if this pull request introduces a breaking change

## What does this implement/fix? Explain your changes.

Prepare this package for release!

### CODE REVIEW

**Review by emekdahl-palmetto — APPROVED** [review]
we're github official :)

File: setup.py [hunk 1]
```diff
@@ -24,12 +24,11 @@
...
     author='Palmetto - Data & Analytics team',
     author_email='data-analytics-team@palmetto.com',
-    url='',
+    url='https://github.com/palmetto/palm-dbt',
     packages=find_namespace_packages(include=['palm', 'palm.*']),
     package_data={'': ['*.md', '*.sql', '*.yaml', '*.yml']},
-    include_package_data=True,
     install_requires=[
-		'palm>=2.0.0'
+		'palmcli>=2.0.0'
```
Comment (mrogers-palmetto) [inline setup.py:31]: What about in requirements.txt?

**Review by mrogers-palmetto — APPROVED** [review]

**Review by emekdahl-palmetto — APPROVED** [review]
bump question about requirements

File: setup.py (hunk 1 above)
Comment (jakeberesford-palmetto) [reply setup.py:31]: thanks!

**Pull request merged**